### PR TITLE
JP-2916: Add Gaia DR3 to available catalogs in tweakreg

### LINKED ISSUE #7217: Add Gaia DR3 to available catalogs in tweakreg
_Issue [JP-2916](https://jira.stsci.edu/browse/JP-2916) was created on JIRA by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse):_

Creating this ticket to track the user request in [github #&#x2060;7007](https://github.com/spacetelescope/jwst/issues/7007), which is asking for the Gaia DR3 catalog to be added to the available options for aligning to Gaia.  Need to get INS thoughts on priority.

### PR DESCRIPTION
**Pull request by s-goldman**
Resolves [JP-2916](https://jira.stsci.edu/browse/JP-2916)

Closes #7217 

This PR adds GAIADR3 to the available catalog options in jwst/tweakreg. The changes pass the jwst/resample and jwst/tweakreg pytests (when DR3 set as default), but no further testing has been done. 

GAIADR3 has not yet been enabled as the default.

**Checklist for maintainers**
- [x] added entry in `CHANGES.rst` within the relevant release section
- [ ] updated or added relevant tests
- [x] updated relevant documentation
- [x] added relevant milestone
- [x] added relevant label(s)
- [ ] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)
- [ ] Make sure the JIRA ticket is [resolved properly](https://github.com/spacetelescope/jwst/wiki/How-to-resolve-JIRA-issues)

### CODE REVIEW

**Review by hbushouse — COMMENTED** [review]
Looks simple and reasonable enough.

Needs a change log entry and needs to have the various places in https://github.com/spacetelescope/jwst/blob/master/docs/jwst/tweakreg/README.rst where GAIADR<n> is mentioned updated to include DR3.

**Comment by s-goldman** [comment]
> Looks simple and reasonable enough.
> 
> Needs a change log entry and needs to have the various places in https://github.com/spacetelescope/jwst/blob/master/docs/jwst/tweakreg/README.rst where GAIADR is mentioned updated to include DR3.

Those have now been added.

File: CHANGES.rst [hunk 1]
```diff
@@ -154,6 +154,9 @@ tweakreg
   another. It is an analog of the ``tweakback`` task in the
   ``drizzlepac``. [#7573, #7591]
 
+- Added the 'GAIADR3' catalog to the available options for alignment; 
+  this has been enabled as the default option [#7217].
```
Comment (hbushouse) [inline CHANGES.rst:158]:
```suggestion
  this has been enabled as the default option [#7611].
```
The number referenced in the change log entry is the PR number, not the JP ticket number.

**Review by hbushouse — APPROVED** [review]
Looks good!

**Pull request merged**